foundryup/foundryup

```diff
@@ -171,11 +171,7 @@ main() {
 
     # Optionally download the manuals
     if check_cmd tar; then
-      say "downloading manpages"
-      mkdir -p "$FOUNDRY_MAN_DIR"
-      download "$MAN_TARBALL_URL" | tar -xzC "$FOUNDRY_MAN_DIR"
```

Comment:
Why do we remove this?

Reply:
because we don't build the manpages in release.yaml